2015-03-13 23:01:26,159 - [INFO] - from play in pool-6-thread-4 
Listening for HTTP on /0:0:0:0:0:0:0:0:9000

2015-03-13 23:01:33,975 - [INFO] - from play in play-internal-execution-context-1 
datasource [jdbc:h2:mem:play;DB_CLOSE_DELAY=-1] bound to JNDI as DefaultDS

2015-03-13 23:01:34,536 - [INFO] - from play in play-internal-execution-context-1 
database [default] connected at jdbc:h2:mem:play

2015-03-13 23:01:38,848 - [INFO] - from play in play-internal-execution-context-1 
Application started (Dev)

2015-03-13 23:03:32,966 - [INFO] - from play in play-internal-execution-context-5 
datasource [jdbc:h2:mem:play;DB_CLOSE_DELAY=-1] bound to JNDI as DefaultDS

2015-03-13 23:03:32,974 - [INFO] - from play in play-internal-execution-context-5 
database [default] connected at jdbc:h2:mem:play

2015-03-13 23:03:33,561 - [INFO] - from play in play-internal-execution-context-5 
Application started (Dev)

2015-03-13 23:04:29,836 - [INFO] - from play in play-internal-execution-context-5 
datasource [jdbc:h2:mem:play;DB_CLOSE_DELAY=-1] bound to JNDI as DefaultDS

2015-03-13 23:04:29,843 - [INFO] - from play in play-internal-execution-context-5 
database [default] connected at jdbc:h2:mem:play

2015-03-13 23:04:30,559 - [INFO] - from play in play-internal-execution-context-5 
Application started (Dev)

2015-03-13 23:05:41,299 - [INFO] - from play in play-internal-execution-context-5 
datasource [jdbc:h2:mem:play;DB_CLOSE_DELAY=-1] bound to JNDI as DefaultDS

2015-03-13 23:05:41,306 - [INFO] - from play in play-internal-execution-context-5 
database [default] connected at jdbc:h2:mem:play

2015-03-13 23:05:41,783 - [INFO] - from play in play-internal-execution-context-5 
Application started (Dev)

2015-03-13 23:06:13,606 - [INFO] - from play in play-internal-execution-context-5 
datasource [jdbc:h2:mem:play;DB_CLOSE_DELAY=-1] bound to JNDI as DefaultDS

2015-03-13 23:06:13,613 - [INFO] - from play in play-internal-execution-context-5 
database [default] connected at jdbc:h2:mem:play

2015-03-13 23:06:14,074 - [INFO] - from play in play-internal-execution-context-5 
Application started (Dev)

2015-03-13 23:08:41,615 - [INFO] - from play in play-internal-execution-context-5 
datasource [jdbc:h2:mem:play;DB_CLOSE_DELAY=-1] bound to JNDI as DefaultDS

2015-03-13 23:08:41,621 - [INFO] - from play in play-internal-execution-context-5 
database [default] connected at jdbc:h2:mem:play

2015-03-13 23:08:42,102 - [INFO] - from play in play-internal-execution-context-5 
Application started (Dev)

2015-03-13 23:09:27,484 - [INFO] - from play in play-internal-execution-context-12 
datasource [jdbc:h2:mem:play;DB_CLOSE_DELAY=-1] bound to JNDI as DefaultDS

2015-03-13 23:09:27,493 - [INFO] - from play in play-internal-execution-context-12 
database [default] connected at jdbc:h2:mem:play

2015-03-13 23:09:28,029 - [INFO] - from play in play-internal-execution-context-12 
Application started (Dev)

2015-03-13 23:11:13,415 - [INFO] - from play in play-internal-execution-context-12 
datasource [jdbc:h2:mem:play;DB_CLOSE_DELAY=-1] bound to JNDI as DefaultDS

2015-03-13 23:11:13,422 - [INFO] - from play in play-internal-execution-context-12 
database [default] connected at jdbc:h2:mem:play

2015-03-13 23:11:13,910 - [INFO] - from play in play-internal-execution-context-12 
Application started (Dev)

2015-03-13 23:12:43,810 - [INFO] - from play in play-internal-execution-context-12 
datasource [jdbc:h2:mem:play;DB_CLOSE_DELAY=-1] bound to JNDI as DefaultDS

2015-03-13 23:12:43,818 - [INFO] - from play in play-internal-execution-context-12 
database [default] connected at jdbc:h2:mem:play

2015-03-13 23:12:44,448 - [INFO] - from play in play-internal-execution-context-12 
Application started (Dev)

2015-03-13 23:13:34,215 - [INFO] - from play in play-internal-execution-context-12 
datasource [jdbc:h2:mem:play;DB_CLOSE_DELAY=-1] bound to JNDI as DefaultDS

2015-03-13 23:13:34,221 - [INFO] - from play in play-internal-execution-context-12 
database [default] connected at jdbc:h2:mem:play

2015-03-13 23:13:34,755 - [INFO] - from play in play-internal-execution-context-12 
Application started (Dev)

2015-03-13 23:14:12,041 - [ERROR] - from application in New I/O worker #6 


! @6lf2olm2m - Internal server error, for (GET) [/] ->

play.PlayExceptions$CompilationException: Compilation error[not found: value tema]
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.1.jar:na]
	at play.PlayReload$$anonfun$taskFailureHandler$1.apply(PlayReload.scala:49) ~[na:na]
	at play.PlayReload$$anonfun$taskFailureHandler$1.apply(PlayReload.scala:44) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.1.jar:na]
	at play.PlayReload$.taskFailureHandler(PlayReload.scala:44) ~[na:na]
	at play.PlayReload$.compileFailure(PlayReload.scala:40) ~[na:na]
	at play.PlayReload$$anonfun$compile$1.apply(PlayReload.scala:17) ~[na:na]
	at play.PlayReload$$anonfun$compile$1.apply(PlayReload.scala:17) ~[na:na]
	at scala.util.Either$LeftProjection.map(Either.scala:377) ~[scala-library-2.11.1.jar:na]
	at play.PlayReload$.compile(PlayReload.scala:17) ~[na:na]
	at play.PlayRun$$anonfun$playRunTask$1$$anonfun$apply$2$$anonfun$apply$3$$anonfun$2.apply(PlayRun.scala:61) ~[na:na]
	at play.PlayRun$$anonfun$playRunTask$1$$anonfun$apply$2$$anonfun$apply$3$$anonfun$2.apply(PlayRun.scala:61) ~[na:na]
	at play.runsupport.Reloader.reload(Reloader.scala:295) ~[na:na]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:122) ~[play_2.11-2.3.8.jar:2.3.8]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:120) ~[play_2.11-2.3.8.jar:2.3.8]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) ~[scala-library-2.11.1.jar:na]

2015-03-13 23:14:34,019 - [INFO] - from play in play-internal-execution-context-12 
datasource [jdbc:h2:mem:play;DB_CLOSE_DELAY=-1] bound to JNDI as DefaultDS

2015-03-13 23:14:34,026 - [INFO] - from play in play-internal-execution-context-12 
database [default] connected at jdbc:h2:mem:play

2015-03-13 23:14:34,538 - [INFO] - from play in play-internal-execution-context-12 
Application started (Dev)

2015-03-13 23:16:02,460 - [INFO] - from play in play-internal-execution-context-12 
datasource [jdbc:h2:mem:play;DB_CLOSE_DELAY=-1] bound to JNDI as DefaultDS

2015-03-13 23:16:02,469 - [INFO] - from play in play-internal-execution-context-12 
database [default] connected at jdbc:h2:mem:play

2015-03-13 23:16:03,004 - [INFO] - from play in play-internal-execution-context-12 
Application started (Dev)

2015-03-13 23:16:26,066 - [INFO] - from play in play-internal-execution-context-25 
datasource [jdbc:h2:mem:play;DB_CLOSE_DELAY=-1] bound to JNDI as DefaultDS

2015-03-13 23:16:26,073 - [INFO] - from play in play-internal-execution-context-25 
database [default] connected at jdbc:h2:mem:play

2015-03-13 23:16:26,639 - [INFO] - from play in play-internal-execution-context-25 
Application started (Dev)

2015-03-13 23:17:21,300 - [INFO] - from play in play-internal-execution-context-26 
datasource [jdbc:h2:mem:play;DB_CLOSE_DELAY=-1] bound to JNDI as DefaultDS

2015-03-13 23:17:21,306 - [INFO] - from play in play-internal-execution-context-26 
database [default] connected at jdbc:h2:mem:play

2015-03-13 23:17:21,736 - [INFO] - from play in play-internal-execution-context-26 
Application started (Dev)

2015-03-13 23:18:11,832 - [INFO] - from play in play-internal-execution-context-26 
datasource [jdbc:h2:mem:play;DB_CLOSE_DELAY=-1] bound to JNDI as DefaultDS

2015-03-13 23:18:11,839 - [INFO] - from play in play-internal-execution-context-26 
database [default] connected at jdbc:h2:mem:play

2015-03-13 23:18:12,288 - [INFO] - from play in play-internal-execution-context-26 
Application started (Dev)

2015-03-13 23:18:37,915 - [INFO] - from play in play-internal-execution-context-26 
datasource [jdbc:h2:mem:play;DB_CLOSE_DELAY=-1] bound to JNDI as DefaultDS

2015-03-13 23:18:37,922 - [INFO] - from play in play-internal-execution-context-26 
database [default] connected at jdbc:h2:mem:play

2015-03-13 23:18:38,375 - [INFO] - from play in play-internal-execution-context-26 
Application started (Dev)

2015-03-13 23:18:58,525 - [INFO] - from play in play-internal-execution-context-26 
datasource [jdbc:h2:mem:play;DB_CLOSE_DELAY=-1] bound to JNDI as DefaultDS

2015-03-13 23:18:58,532 - [INFO] - from play in play-internal-execution-context-26 
database [default] connected at jdbc:h2:mem:play

2015-03-13 23:18:58,976 - [INFO] - from play in play-internal-execution-context-26 
Application started (Dev)

2015-03-13 23:19:20,240 - [INFO] - from play in play-internal-execution-context-26 
datasource [jdbc:h2:mem:play;DB_CLOSE_DELAY=-1] bound to JNDI as DefaultDS

2015-03-13 23:19:20,250 - [INFO] - from play in play-internal-execution-context-26 
database [default] connected at jdbc:h2:mem:play

2015-03-13 23:19:20,697 - [INFO] - from play in play-internal-execution-context-26 
Application started (Dev)

2015-03-13 23:20:05,741 - [INFO] - from play in play-internal-execution-context-26 
datasource [jdbc:h2:mem:play;DB_CLOSE_DELAY=-1] bound to JNDI as DefaultDS

2015-03-13 23:20:05,750 - [INFO] - from play in play-internal-execution-context-26 
database [default] connected at jdbc:h2:mem:play

2015-03-13 23:20:06,269 - [INFO] - from play in play-internal-execution-context-26 
Application started (Dev)

2015-03-13 23:20:31,915 - [INFO] - from play in play-internal-execution-context-26 
datasource [jdbc:h2:mem:play;DB_CLOSE_DELAY=-1] bound to JNDI as DefaultDS

2015-03-13 23:20:31,922 - [INFO] - from play in play-internal-execution-context-26 
database [default] connected at jdbc:h2:mem:play

2015-03-13 23:20:32,365 - [INFO] - from play in play-internal-execution-context-26 
Application started (Dev)

2015-03-13 23:21:09,285 - [INFO] - from play in play-internal-execution-context-42 
datasource [jdbc:h2:mem:play;DB_CLOSE_DELAY=-1] bound to JNDI as DefaultDS

2015-03-13 23:21:09,292 - [INFO] - from play in play-internal-execution-context-42 
database [default] connected at jdbc:h2:mem:play

2015-03-13 23:21:09,666 - [INFO] - from play in play-internal-execution-context-42 
Application started (Dev)

2015-03-13 23:21:57,764 - [INFO] - from play in play-internal-execution-context-42 
datasource [jdbc:h2:mem:play;DB_CLOSE_DELAY=-1] bound to JNDI as DefaultDS

2015-03-13 23:21:57,772 - [INFO] - from play in play-internal-execution-context-42 
database [default] connected at jdbc:h2:mem:play

2015-03-13 23:21:58,375 - [INFO] - from play in play-internal-execution-context-42 
Application started (Dev)

2015-03-13 23:22:37,313 - [INFO] - from play in play-internal-execution-context-46 
datasource [jdbc:h2:mem:play;DB_CLOSE_DELAY=-1] bound to JNDI as DefaultDS

2015-03-13 23:22:37,320 - [INFO] - from play in play-internal-execution-context-46 
database [default] connected at jdbc:h2:mem:play

2015-03-13 23:22:37,839 - [INFO] - from play in play-internal-execution-context-46 
Application started (Dev)

2015-03-13 23:23:31,326 - [INFO] - from play in play-internal-execution-context-46 
datasource [jdbc:h2:mem:play;DB_CLOSE_DELAY=-1] bound to JNDI as DefaultDS

2015-03-13 23:23:31,333 - [INFO] - from play in play-internal-execution-context-46 
database [default] connected at jdbc:h2:mem:play

2015-03-13 23:23:31,777 - [INFO] - from play in play-internal-execution-context-46 
Application started (Dev)

2015-03-13 23:24:39,399 - [INFO] - from play in play-internal-execution-context-46 
datasource [jdbc:h2:mem:play;DB_CLOSE_DELAY=-1] bound to JNDI as DefaultDS

2015-03-13 23:24:39,405 - [INFO] - from play in play-internal-execution-context-46 
database [default] connected at jdbc:h2:mem:play

2015-03-13 23:24:39,929 - [INFO] - from play in play-internal-execution-context-46 
Application started (Dev)

2015-03-13 23:25:12,778 - [INFO] - from play in play-internal-execution-context-46 
datasource [jdbc:h2:mem:play;DB_CLOSE_DELAY=-1] bound to JNDI as DefaultDS

2015-03-13 23:25:12,788 - [INFO] - from play in play-internal-execution-context-46 
database [default] connected at jdbc:h2:mem:play

2015-03-13 23:25:13,215 - [INFO] - from play in play-internal-execution-context-46 
Application started (Dev)

2015-03-13 23:25:41,578 - [INFO] - from play in play-internal-execution-context-57 
datasource [jdbc:h2:mem:play;DB_CLOSE_DELAY=-1] bound to JNDI as DefaultDS

2015-03-13 23:25:41,585 - [INFO] - from play in play-internal-execution-context-57 
database [default] connected at jdbc:h2:mem:play

2015-03-13 23:25:42,088 - [INFO] - from play in play-internal-execution-context-57 
Application started (Dev)

2015-03-13 23:27:19,719 - [INFO] - from play in play-internal-execution-context-54 
datasource [jdbc:h2:mem:play;DB_CLOSE_DELAY=-1] bound to JNDI as DefaultDS

2015-03-13 23:27:19,726 - [INFO] - from play in play-internal-execution-context-54 
database [default] connected at jdbc:h2:mem:play

2015-03-13 23:27:20,147 - [INFO] - from play in play-internal-execution-context-54 
Application started (Dev)

2015-03-13 23:27:45,619 - [INFO] - from play in play-internal-execution-context-54 
datasource [jdbc:h2:mem:play;DB_CLOSE_DELAY=-1] bound to JNDI as DefaultDS

2015-03-13 23:27:45,628 - [INFO] - from play in play-internal-execution-context-54 
database [default] connected at jdbc:h2:mem:play

2015-03-13 23:27:46,126 - [INFO] - from play in play-internal-execution-context-54 
Application started (Dev)

2015-03-13 23:28:11,506 - [INFO] - from play in play-internal-execution-context-54 
datasource [jdbc:h2:mem:play;DB_CLOSE_DELAY=-1] bound to JNDI as DefaultDS

2015-03-13 23:28:11,513 - [INFO] - from play in play-internal-execution-context-54 
database [default] connected at jdbc:h2:mem:play

2015-03-13 23:28:11,935 - [INFO] - from play in play-internal-execution-context-54 
Application started (Dev)

2015-03-13 23:28:55,389 - [INFO] - from play in play-internal-execution-context-54 
datasource [jdbc:h2:mem:play;DB_CLOSE_DELAY=-1] bound to JNDI as DefaultDS

2015-03-13 23:28:55,397 - [INFO] - from play in play-internal-execution-context-54 
database [default] connected at jdbc:h2:mem:play

2015-03-13 23:28:55,912 - [INFO] - from play in play-internal-execution-context-54 
Application started (Dev)

2015-03-13 23:29:26,577 - [INFO] - from play in play-internal-execution-context-54 
datasource [jdbc:h2:mem:play;DB_CLOSE_DELAY=-1] bound to JNDI as DefaultDS

2015-03-13 23:29:26,584 - [INFO] - from play in play-internal-execution-context-54 
database [default] connected at jdbc:h2:mem:play

2015-03-13 23:29:27,007 - [INFO] - from play in play-internal-execution-context-54 
Application started (Dev)

2015-03-13 23:29:49,723 - [INFO] - from play in play-internal-execution-context-54 
datasource [jdbc:h2:mem:play;DB_CLOSE_DELAY=-1] bound to JNDI as DefaultDS

2015-03-13 23:29:49,730 - [INFO] - from play in play-internal-execution-context-54 
database [default] connected at jdbc:h2:mem:play

2015-03-13 23:29:50,169 - [INFO] - from play in play-internal-execution-context-54 
Application started (Dev)

2015-03-13 23:31:23,954 - [INFO] - from play in play-internal-execution-context-54 
datasource [jdbc:h2:mem:play;DB_CLOSE_DELAY=-1] bound to JNDI as DefaultDS

2015-03-13 23:31:23,962 - [INFO] - from play in play-internal-execution-context-54 
database [default] connected at jdbc:h2:mem:play

2015-03-13 23:31:24,438 - [INFO] - from play in play-internal-execution-context-54 
Application started (Dev)

2015-03-13 23:33:26,761 - [ERROR] - from application in New I/O worker #4 


! @6lf319e7n - Internal server error, for (GET) [/] ->

play.PlayExceptions$CompilationException: Compilation error[Expected ending '}']
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.1.jar:na]
	at play.PlayReload$$anonfun$taskFailureHandler$1.apply(PlayReload.scala:49) ~[na:na]
	at play.PlayReload$$anonfun$taskFailureHandler$1.apply(PlayReload.scala:44) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.1.jar:na]
	at play.PlayReload$.taskFailureHandler(PlayReload.scala:44) ~[na:na]
	at play.PlayReload$.compileFailure(PlayReload.scala:40) ~[na:na]
	at play.PlayReload$$anonfun$compile$1.apply(PlayReload.scala:17) ~[na:na]
	at play.PlayReload$$anonfun$compile$1.apply(PlayReload.scala:17) ~[na:na]
	at scala.util.Either$LeftProjection.map(Either.scala:377) ~[scala-library-2.11.1.jar:na]
	at play.PlayReload$.compile(PlayReload.scala:17) ~[na:na]
	at play.PlayRun$$anonfun$playRunTask$1$$anonfun$apply$2$$anonfun$apply$3$$anonfun$2.apply(PlayRun.scala:61) ~[na:na]
	at play.PlayRun$$anonfun$playRunTask$1$$anonfun$apply$2$$anonfun$apply$3$$anonfun$2.apply(PlayRun.scala:61) ~[na:na]
	at play.runsupport.Reloader.reload(Reloader.scala:295) ~[na:na]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:122) ~[play_2.11-2.3.8.jar:2.3.8]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:120) ~[play_2.11-2.3.8.jar:2.3.8]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) ~[scala-library-2.11.1.jar:na]

2015-03-13 23:35:01,066 - [ERROR] - from application in New I/O worker #4 


! @6lf31enl5 - Internal server error, for (GET) [/] ->

play.PlayExceptions$CompilationException: Compilation error[Expected ending '}']
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.1.jar:na]
	at play.PlayReload$$anonfun$taskFailureHandler$1.apply(PlayReload.scala:49) ~[na:na]
	at play.PlayReload$$anonfun$taskFailureHandler$1.apply(PlayReload.scala:44) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.1.jar:na]
	at play.PlayReload$.taskFailureHandler(PlayReload.scala:44) ~[na:na]
	at play.PlayReload$.compileFailure(PlayReload.scala:40) ~[na:na]
	at play.PlayReload$$anonfun$compile$1.apply(PlayReload.scala:17) ~[na:na]
	at play.PlayReload$$anonfun$compile$1.apply(PlayReload.scala:17) ~[na:na]
	at scala.util.Either$LeftProjection.map(Either.scala:377) ~[scala-library-2.11.1.jar:na]
	at play.PlayReload$.compile(PlayReload.scala:17) ~[na:na]
	at play.PlayRun$$anonfun$playRunTask$1$$anonfun$apply$2$$anonfun$apply$3$$anonfun$2.apply(PlayRun.scala:61) ~[na:na]
	at play.PlayRun$$anonfun$playRunTask$1$$anonfun$apply$2$$anonfun$apply$3$$anonfun$2.apply(PlayRun.scala:61) ~[na:na]
	at play.runsupport.Reloader.reload(Reloader.scala:295) ~[na:na]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:122) ~[play_2.11-2.3.8.jar:2.3.8]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:120) ~[play_2.11-2.3.8.jar:2.3.8]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) ~[scala-library-2.11.1.jar:na]

2015-03-13 23:35:08,462 - [ERROR] - from application in New I/O worker #4 


! @6lf31f8jb - Internal server error, for (GET) [/] ->

play.PlayExceptions$CompilationException: Compilation error[Expected ending '}']
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.1.jar:na]
	at play.PlayReload$$anonfun$taskFailureHandler$1.apply(PlayReload.scala:49) ~[na:na]
	at play.PlayReload$$anonfun$taskFailureHandler$1.apply(PlayReload.scala:44) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.1.jar:na]
	at play.PlayReload$.taskFailureHandler(PlayReload.scala:44) ~[na:na]
	at play.PlayReload$.compileFailure(PlayReload.scala:40) ~[na:na]
	at play.PlayReload$$anonfun$compile$1.apply(PlayReload.scala:17) ~[na:na]
	at play.PlayReload$$anonfun$compile$1.apply(PlayReload.scala:17) ~[na:na]
	at scala.util.Either$LeftProjection.map(Either.scala:377) ~[scala-library-2.11.1.jar:na]
	at play.PlayReload$.compile(PlayReload.scala:17) ~[na:na]
	at play.PlayRun$$anonfun$playRunTask$1$$anonfun$apply$2$$anonfun$apply$3$$anonfun$2.apply(PlayRun.scala:61) ~[na:na]
	at play.PlayRun$$anonfun$playRunTask$1$$anonfun$apply$2$$anonfun$apply$3$$anonfun$2.apply(PlayRun.scala:61) ~[na:na]
	at play.runsupport.Reloader.reload(Reloader.scala:295) ~[na:na]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:122) ~[play_2.11-2.3.8.jar:2.3.8]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:120) ~[play_2.11-2.3.8.jar:2.3.8]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) ~[scala-library-2.11.1.jar:na]

2015-03-13 23:35:11,331 - [ERROR] - from application in New I/O worker #6 


! @6lf31fcpp - Internal server error, for (GET) [/] ->

play.PlayExceptions$CompilationException: Compilation error[Expected ending '}']
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.1.jar:na]
	at play.PlayReload$$anonfun$taskFailureHandler$1.apply(PlayReload.scala:49) ~[na:na]
	at play.PlayReload$$anonfun$taskFailureHandler$1.apply(PlayReload.scala:44) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.1.jar:na]
	at play.PlayReload$.taskFailureHandler(PlayReload.scala:44) ~[na:na]
	at play.PlayReload$.compileFailure(PlayReload.scala:40) ~[na:na]
	at play.PlayReload$$anonfun$compile$1.apply(PlayReload.scala:17) ~[na:na]
	at play.PlayReload$$anonfun$compile$1.apply(PlayReload.scala:17) ~[na:na]
	at scala.util.Either$LeftProjection.map(Either.scala:377) ~[scala-library-2.11.1.jar:na]
	at play.PlayReload$.compile(PlayReload.scala:17) ~[na:na]
	at play.PlayRun$$anonfun$playRunTask$1$$anonfun$apply$2$$anonfun$apply$3$$anonfun$2.apply(PlayRun.scala:61) ~[na:na]
	at play.PlayRun$$anonfun$playRunTask$1$$anonfun$apply$2$$anonfun$apply$3$$anonfun$2.apply(PlayRun.scala:61) ~[na:na]
	at play.runsupport.Reloader.reload(Reloader.scala:295) ~[na:na]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:122) ~[play_2.11-2.3.8.jar:2.3.8]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:120) ~[play_2.11-2.3.8.jar:2.3.8]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) ~[scala-library-2.11.1.jar:na]

2015-03-13 23:35:33,373 - [ERROR] - from application in New I/O worker #6 


! @6lf31gjfm - Internal server error, for (GET) [/] ->

play.PlayExceptions$CompilationException: Compilation error[Expected ending '}']
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.1.jar:na]
	at play.PlayReload$$anonfun$taskFailureHandler$1.apply(PlayReload.scala:49) ~[na:na]
	at play.PlayReload$$anonfun$taskFailureHandler$1.apply(PlayReload.scala:44) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.1.jar:na]
	at play.PlayReload$.taskFailureHandler(PlayReload.scala:44) ~[na:na]
	at play.PlayReload$.compileFailure(PlayReload.scala:40) ~[na:na]
	at play.PlayReload$$anonfun$compile$1.apply(PlayReload.scala:17) ~[na:na]
	at play.PlayReload$$anonfun$compile$1.apply(PlayReload.scala:17) ~[na:na]
	at scala.util.Either$LeftProjection.map(Either.scala:377) ~[scala-library-2.11.1.jar:na]
	at play.PlayReload$.compile(PlayReload.scala:17) ~[na:na]
	at play.PlayRun$$anonfun$playRunTask$1$$anonfun$apply$2$$anonfun$apply$3$$anonfun$2.apply(PlayRun.scala:61) ~[na:na]
	at play.PlayRun$$anonfun$playRunTask$1$$anonfun$apply$2$$anonfun$apply$3$$anonfun$2.apply(PlayRun.scala:61) ~[na:na]
	at play.runsupport.Reloader.reload(Reloader.scala:295) ~[na:na]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:122) ~[play_2.11-2.3.8.jar:2.3.8]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:120) ~[play_2.11-2.3.8.jar:2.3.8]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) ~[scala-library-2.11.1.jar:na]

2015-03-13 23:35:39,576 - [ERROR] - from application in New I/O worker #6 


! @6lf31h2k9 - Internal server error, for (GET) [/] ->

play.PlayExceptions$CompilationException: Compilation error[Expected ending '}']
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.1.jar:na]
	at play.PlayReload$$anonfun$taskFailureHandler$1.apply(PlayReload.scala:49) ~[na:na]
	at play.PlayReload$$anonfun$taskFailureHandler$1.apply(PlayReload.scala:44) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.1.jar:na]
	at play.PlayReload$.taskFailureHandler(PlayReload.scala:44) ~[na:na]
	at play.PlayReload$.compileFailure(PlayReload.scala:40) ~[na:na]
	at play.PlayReload$$anonfun$compile$1.apply(PlayReload.scala:17) ~[na:na]
	at play.PlayReload$$anonfun$compile$1.apply(PlayReload.scala:17) ~[na:na]
	at scala.util.Either$LeftProjection.map(Either.scala:377) ~[scala-library-2.11.1.jar:na]
	at play.PlayReload$.compile(PlayReload.scala:17) ~[na:na]
	at play.PlayRun$$anonfun$playRunTask$1$$anonfun$apply$2$$anonfun$apply$3$$anonfun$2.apply(PlayRun.scala:61) ~[na:na]
	at play.PlayRun$$anonfun$playRunTask$1$$anonfun$apply$2$$anonfun$apply$3$$anonfun$2.apply(PlayRun.scala:61) ~[na:na]
	at play.runsupport.Reloader.reload(Reloader.scala:295) ~[na:na]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:122) ~[play_2.11-2.3.8.jar:2.3.8]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:120) ~[play_2.11-2.3.8.jar:2.3.8]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) ~[scala-library-2.11.1.jar:na]

2015-03-13 23:36:39,606 - [ERROR] - from application in New I/O worker #6 


! @6lf31kdf4 - Internal server error, for (GET) [/] ->

play.PlayExceptions$CompilationException: Compilation error[Expected ending '}']
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.1.jar:na]
	at play.PlayReload$$anonfun$taskFailureHandler$1.apply(PlayReload.scala:49) ~[na:na]
	at play.PlayReload$$anonfun$taskFailureHandler$1.apply(PlayReload.scala:44) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.1.jar:na]
	at play.PlayReload$.taskFailureHandler(PlayReload.scala:44) ~[na:na]
	at play.PlayReload$.compileFailure(PlayReload.scala:40) ~[na:na]
	at play.PlayReload$$anonfun$compile$1.apply(PlayReload.scala:17) ~[na:na]
	at play.PlayReload$$anonfun$compile$1.apply(PlayReload.scala:17) ~[na:na]
	at scala.util.Either$LeftProjection.map(Either.scala:377) ~[scala-library-2.11.1.jar:na]
	at play.PlayReload$.compile(PlayReload.scala:17) ~[na:na]
	at play.PlayRun$$anonfun$playRunTask$1$$anonfun$apply$2$$anonfun$apply$3$$anonfun$2.apply(PlayRun.scala:61) ~[na:na]
	at play.PlayRun$$anonfun$playRunTask$1$$anonfun$apply$2$$anonfun$apply$3$$anonfun$2.apply(PlayRun.scala:61) ~[na:na]
	at play.runsupport.Reloader.reload(Reloader.scala:295) ~[na:na]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:122) ~[play_2.11-2.3.8.jar:2.3.8]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:120) ~[play_2.11-2.3.8.jar:2.3.8]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) ~[scala-library-2.11.1.jar:na]

2015-03-13 23:36:46,246 - [ERROR] - from application in New I/O worker #6 


! @6lf31knag - Internal server error, for (GET) [/] ->

play.PlayExceptions$CompilationException: Compilation error[Expected ending '}']
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.1.jar:na]
	at play.PlayReload$$anonfun$taskFailureHandler$1.apply(PlayReload.scala:49) ~[na:na]
	at play.PlayReload$$anonfun$taskFailureHandler$1.apply(PlayReload.scala:44) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.1.jar:na]
	at play.PlayReload$.taskFailureHandler(PlayReload.scala:44) ~[na:na]
	at play.PlayReload$.compileFailure(PlayReload.scala:40) ~[na:na]
	at play.PlayReload$$anonfun$compile$1.apply(PlayReload.scala:17) ~[na:na]
	at play.PlayReload$$anonfun$compile$1.apply(PlayReload.scala:17) ~[na:na]
	at scala.util.Either$LeftProjection.map(Either.scala:377) ~[scala-library-2.11.1.jar:na]
	at play.PlayReload$.compile(PlayReload.scala:17) ~[na:na]
	at play.PlayRun$$anonfun$playRunTask$1$$anonfun$apply$2$$anonfun$apply$3$$anonfun$2.apply(PlayRun.scala:61) ~[na:na]
	at play.PlayRun$$anonfun$playRunTask$1$$anonfun$apply$2$$anonfun$apply$3$$anonfun$2.apply(PlayRun.scala:61) ~[na:na]
	at play.runsupport.Reloader.reload(Reloader.scala:295) ~[na:na]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:122) ~[play_2.11-2.3.8.jar:2.3.8]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:120) ~[play_2.11-2.3.8.jar:2.3.8]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) ~[scala-library-2.11.1.jar:na]

2015-03-13 23:36:59,057 - [ERROR] - from application in New I/O worker #6 


! @6lf31lg99 - Internal server error, for (GET) [/] ->

play.PlayExceptions$CompilationException: Compilation error[Expected '}' but found 'EOF']
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.1.jar:na]
	at play.PlayReload$$anonfun$taskFailureHandler$1.apply(PlayReload.scala:49) ~[na:na]
	at play.PlayReload$$anonfun$taskFailureHandler$1.apply(PlayReload.scala:44) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.1.jar:na]
	at play.PlayReload$.taskFailureHandler(PlayReload.scala:44) ~[na:na]
	at play.PlayReload$.compileFailure(PlayReload.scala:40) ~[na:na]
	at play.PlayReload$$anonfun$compile$1.apply(PlayReload.scala:17) ~[na:na]
	at play.PlayReload$$anonfun$compile$1.apply(PlayReload.scala:17) ~[na:na]
	at scala.util.Either$LeftProjection.map(Either.scala:377) ~[scala-library-2.11.1.jar:na]
	at play.PlayReload$.compile(PlayReload.scala:17) ~[na:na]
	at play.PlayRun$$anonfun$playRunTask$1$$anonfun$apply$2$$anonfun$apply$3$$anonfun$2.apply(PlayRun.scala:61) ~[na:na]
	at play.PlayRun$$anonfun$playRunTask$1$$anonfun$apply$2$$anonfun$apply$3$$anonfun$2.apply(PlayRun.scala:61) ~[na:na]
	at play.runsupport.Reloader.reload(Reloader.scala:295) ~[na:na]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:122) ~[play_2.11-2.3.8.jar:2.3.8]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:120) ~[play_2.11-2.3.8.jar:2.3.8]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) ~[scala-library-2.11.1.jar:na]

2015-03-13 23:37:25,769 - [INFO] - from play in play-internal-execution-context-73 
datasource [jdbc:h2:mem:play;DB_CLOSE_DELAY=-1] bound to JNDI as DefaultDS

2015-03-13 23:37:25,775 - [INFO] - from play in play-internal-execution-context-73 
database [default] connected at jdbc:h2:mem:play

2015-03-13 23:37:26,215 - [INFO] - from play in play-internal-execution-context-73 
Application started (Dev)

2015-03-13 23:37:58,076 - [ERROR] - from application in New I/O worker #6 


! @6lf31oph7 - Internal server error, for (GET) [/] ->

play.PlayExceptions$CompilationException: Compilation error[Expected ending '}']
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.1.jar:na]
	at play.PlayReload$$anonfun$taskFailureHandler$1.apply(PlayReload.scala:49) ~[na:na]
	at play.PlayReload$$anonfun$taskFailureHandler$1.apply(PlayReload.scala:44) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.1.jar:na]
	at play.PlayReload$.taskFailureHandler(PlayReload.scala:44) ~[na:na]
	at play.PlayReload$.compileFailure(PlayReload.scala:40) ~[na:na]
	at play.PlayReload$$anonfun$compile$1.apply(PlayReload.scala:17) ~[na:na]
	at play.PlayReload$$anonfun$compile$1.apply(PlayReload.scala:17) ~[na:na]
	at scala.util.Either$LeftProjection.map(Either.scala:377) ~[scala-library-2.11.1.jar:na]
	at play.PlayReload$.compile(PlayReload.scala:17) ~[na:na]
	at play.PlayRun$$anonfun$playRunTask$1$$anonfun$apply$2$$anonfun$apply$3$$anonfun$2.apply(PlayRun.scala:61) ~[na:na]
	at play.PlayRun$$anonfun$playRunTask$1$$anonfun$apply$2$$anonfun$apply$3$$anonfun$2.apply(PlayRun.scala:61) ~[na:na]
	at play.runsupport.Reloader.reload(Reloader.scala:295) ~[na:na]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:122) ~[play_2.11-2.3.8.jar:2.3.8]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:120) ~[play_2.11-2.3.8.jar:2.3.8]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) ~[scala-library-2.11.1.jar:na]

2015-03-13 23:38:18,392 - [ERROR] - from application in New I/O worker #6 


! @6lf3203ii - Internal server error, for (GET) [/] ->

play.PlayExceptions$CompilationException: Compilation error[Expected '}' but found 'EOF']
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.1.jar:na]
	at play.PlayReload$$anonfun$taskFailureHandler$1.apply(PlayReload.scala:49) ~[na:na]
	at play.PlayReload$$anonfun$taskFailureHandler$1.apply(PlayReload.scala:44) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.1.jar:na]
	at play.PlayReload$.taskFailureHandler(PlayReload.scala:44) ~[na:na]
	at play.PlayReload$.compileFailure(PlayReload.scala:40) ~[na:na]
	at play.PlayReload$$anonfun$compile$1.apply(PlayReload.scala:17) ~[na:na]
	at play.PlayReload$$anonfun$compile$1.apply(PlayReload.scala:17) ~[na:na]
	at scala.util.Either$LeftProjection.map(Either.scala:377) ~[scala-library-2.11.1.jar:na]
	at play.PlayReload$.compile(PlayReload.scala:17) ~[na:na]
	at play.PlayRun$$anonfun$playRunTask$1$$anonfun$apply$2$$anonfun$apply$3$$anonfun$2.apply(PlayRun.scala:61) ~[na:na]
	at play.PlayRun$$anonfun$playRunTask$1$$anonfun$apply$2$$anonfun$apply$3$$anonfun$2.apply(PlayRun.scala:61) ~[na:na]
	at play.runsupport.Reloader.reload(Reloader.scala:295) ~[na:na]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:122) ~[play_2.11-2.3.8.jar:2.3.8]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:120) ~[play_2.11-2.3.8.jar:2.3.8]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) ~[scala-library-2.11.1.jar:na]

2015-03-13 23:38:24,183 - [ERROR] - from application in New I/O worker #6 


! @6lf320c7a - Internal server error, for (GET) [/] ->

play.PlayExceptions$CompilationException: Compilation error[Expected '}' but found 'EOF']
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.1.jar:na]
	at play.PlayReload$$anonfun$taskFailureHandler$1.apply(PlayReload.scala:49) ~[na:na]
	at play.PlayReload$$anonfun$taskFailureHandler$1.apply(PlayReload.scala:44) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.1.jar:na]
	at play.PlayReload$.taskFailureHandler(PlayReload.scala:44) ~[na:na]
	at play.PlayReload$.compileFailure(PlayReload.scala:40) ~[na:na]
	at play.PlayReload$$anonfun$compile$1.apply(PlayReload.scala:17) ~[na:na]
	at play.PlayReload$$anonfun$compile$1.apply(PlayReload.scala:17) ~[na:na]
	at scala.util.Either$LeftProjection.map(Either.scala:377) ~[scala-library-2.11.1.jar:na]
	at play.PlayReload$.compile(PlayReload.scala:17) ~[na:na]
	at play.PlayRun$$anonfun$playRunTask$1$$anonfun$apply$2$$anonfun$apply$3$$anonfun$2.apply(PlayRun.scala:61) ~[na:na]
	at play.PlayRun$$anonfun$playRunTask$1$$anonfun$apply$2$$anonfun$apply$3$$anonfun$2.apply(PlayRun.scala:61) ~[na:na]
	at play.runsupport.Reloader.reload(Reloader.scala:295) ~[na:na]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:122) ~[play_2.11-2.3.8.jar:2.3.8]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:120) ~[play_2.11-2.3.8.jar:2.3.8]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) ~[scala-library-2.11.1.jar:na]

2015-03-13 23:38:26,934 - [ERROR] - from application in New I/O worker #3 


! @6lf320g96 - Internal server error, for (GET) [/] ->

play.PlayExceptions$CompilationException: Compilation error[Expected '}' but found 'EOF']
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.1.jar:na]
	at play.PlayReload$$anonfun$taskFailureHandler$1.apply(PlayReload.scala:49) ~[na:na]
	at play.PlayReload$$anonfun$taskFailureHandler$1.apply(PlayReload.scala:44) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.1.jar:na]
	at play.PlayReload$.taskFailureHandler(PlayReload.scala:44) ~[na:na]
	at play.PlayReload$.compileFailure(PlayReload.scala:40) ~[na:na]
	at play.PlayReload$$anonfun$compile$1.apply(PlayReload.scala:17) ~[na:na]
	at play.PlayReload$$anonfun$compile$1.apply(PlayReload.scala:17) ~[na:na]
	at scala.util.Either$LeftProjection.map(Either.scala:377) ~[scala-library-2.11.1.jar:na]
	at play.PlayReload$.compile(PlayReload.scala:17) ~[na:na]
	at play.PlayRun$$anonfun$playRunTask$1$$anonfun$apply$2$$anonfun$apply$3$$anonfun$2.apply(PlayRun.scala:61) ~[na:na]
	at play.PlayRun$$anonfun$playRunTask$1$$anonfun$apply$2$$anonfun$apply$3$$anonfun$2.apply(PlayRun.scala:61) ~[na:na]
	at play.runsupport.Reloader.reload(Reloader.scala:295) ~[na:na]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:122) ~[play_2.11-2.3.8.jar:2.3.8]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:120) ~[play_2.11-2.3.8.jar:2.3.8]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) ~[scala-library-2.11.1.jar:na]

2015-03-13 23:38:56,466 - [INFO] - from play in play-internal-execution-context-74 
datasource [jdbc:h2:mem:play;DB_CLOSE_DELAY=-1] bound to JNDI as DefaultDS

2015-03-13 23:38:56,474 - [INFO] - from play in play-internal-execution-context-74 
database [default] connected at jdbc:h2:mem:play

2015-03-13 23:38:56,932 - [INFO] - from play in play-internal-execution-context-74 
Application started (Dev)

2015-03-13 23:40:09,308 - [INFO] - from play in play-internal-execution-context-74 
datasource [jdbc:h2:mem:play;DB_CLOSE_DELAY=-1] bound to JNDI as DefaultDS

2015-03-13 23:40:09,314 - [INFO] - from play in play-internal-execution-context-74 
database [default] connected at jdbc:h2:mem:play

2015-03-13 23:40:09,744 - [INFO] - from play in play-internal-execution-context-74 
Application started (Dev)

2015-03-13 23:45:45,495 - [INFO] - from play in play-internal-execution-context-74 
datasource [jdbc:h2:mem:play;DB_CLOSE_DELAY=-1] bound to JNDI as DefaultDS

2015-03-13 23:45:45,501 - [INFO] - from play in play-internal-execution-context-74 
database [default] connected at jdbc:h2:mem:play

2015-03-13 23:45:45,927 - [INFO] - from play in play-internal-execution-context-74 
Application started (Dev)

2015-03-13 23:46:16,728 - [INFO] - from play in play-internal-execution-context-81 
datasource [jdbc:h2:mem:play;DB_CLOSE_DELAY=-1] bound to JNDI as DefaultDS

2015-03-13 23:46:16,738 - [INFO] - from play in play-internal-execution-context-81 
database [default] connected at jdbc:h2:mem:play

2015-03-13 23:46:17,165 - [INFO] - from play in play-internal-execution-context-81 
Application started (Dev)

2015-03-13 23:46:47,871 - [INFO] - from play in play-internal-execution-context-81 
datasource [jdbc:h2:mem:play;DB_CLOSE_DELAY=-1] bound to JNDI as DefaultDS

2015-03-13 23:46:47,877 - [INFO] - from play in play-internal-execution-context-81 
database [default] connected at jdbc:h2:mem:play

2015-03-13 23:46:48,322 - [INFO] - from play in play-internal-execution-context-81 
Application started (Dev)

2015-03-13 23:47:32,245 - [INFO] - from play in play-internal-execution-context-86 
datasource [jdbc:h2:mem:play;DB_CLOSE_DELAY=-1] bound to JNDI as DefaultDS

2015-03-13 23:47:32,251 - [INFO] - from play in play-internal-execution-context-86 
database [default] connected at jdbc:h2:mem:play

2015-03-13 23:47:32,734 - [INFO] - from play in play-internal-execution-context-86 
Application started (Dev)

2015-03-13 23:48:00,120 - [INFO] - from play in play-internal-execution-context-86 
datasource [jdbc:h2:mem:play;DB_CLOSE_DELAY=-1] bound to JNDI as DefaultDS

2015-03-13 23:48:00,127 - [INFO] - from play in play-internal-execution-context-86 
database [default] connected at jdbc:h2:mem:play

2015-03-13 23:48:00,564 - [INFO] - from play in play-internal-execution-context-86 
Application started (Dev)

2015-03-13 23:48:33,066 - [INFO] - from play in play-internal-execution-context-86 
datasource [jdbc:h2:mem:play;DB_CLOSE_DELAY=-1] bound to JNDI as DefaultDS

2015-03-13 23:48:33,072 - [INFO] - from play in play-internal-execution-context-86 
database [default] connected at jdbc:h2:mem:play

2015-03-13 23:48:33,490 - [INFO] - from play in play-internal-execution-context-86 
Application started (Dev)

2015-03-13 23:48:58,172 - [INFO] - from play in play-internal-execution-context-86 
datasource [jdbc:h2:mem:play;DB_CLOSE_DELAY=-1] bound to JNDI as DefaultDS

2015-03-13 23:48:58,178 - [INFO] - from play in play-internal-execution-context-86 
database [default] connected at jdbc:h2:mem:play

2015-03-13 23:48:58,600 - [INFO] - from play in play-internal-execution-context-86 
Application started (Dev)

2015-03-13 23:49:45,281 - [INFO] - from play in play-internal-execution-context-94 
datasource [jdbc:h2:mem:play;DB_CLOSE_DELAY=-1] bound to JNDI as DefaultDS

2015-03-13 23:49:45,287 - [INFO] - from play in play-internal-execution-context-94 
database [default] connected at jdbc:h2:mem:play

2015-03-13 23:49:45,624 - [INFO] - from play in play-internal-execution-context-94 
Application started (Dev)

2015-03-13 23:50:21,895 - [INFO] - from play in play-internal-execution-context-94 
datasource [jdbc:h2:mem:play;DB_CLOSE_DELAY=-1] bound to JNDI as DefaultDS

2015-03-13 23:50:21,901 - [INFO] - from play in play-internal-execution-context-94 
database [default] connected at jdbc:h2:mem:play

2015-03-13 23:50:22,348 - [INFO] - from play in play-internal-execution-context-94 
Application started (Dev)

2015-03-13 23:50:58,977 - [INFO] - from play in play-internal-execution-context-94 
datasource [jdbc:h2:mem:play;DB_CLOSE_DELAY=-1] bound to JNDI as DefaultDS

2015-03-13 23:50:58,987 - [INFO] - from play in play-internal-execution-context-94 
database [default] connected at jdbc:h2:mem:play

2015-03-13 23:50:59,457 - [INFO] - from play in play-internal-execution-context-94 
Application started (Dev)

2015-03-13 23:51:27,663 - [INFO] - from play in play-internal-execution-context-94 
datasource [jdbc:h2:mem:play;DB_CLOSE_DELAY=-1] bound to JNDI as DefaultDS

2015-03-13 23:51:27,670 - [INFO] - from play in play-internal-execution-context-94 
database [default] connected at jdbc:h2:mem:play

2015-03-13 23:51:28,119 - [INFO] - from play in play-internal-execution-context-94 
Application started (Dev)

2015-03-13 23:52:04,611 - [INFO] - from play in play-internal-execution-context-94 
datasource [jdbc:h2:mem:play;DB_CLOSE_DELAY=-1] bound to JNDI as DefaultDS

2015-03-13 23:52:04,618 - [INFO] - from play in play-internal-execution-context-94 
database [default] connected at jdbc:h2:mem:play

2015-03-13 23:52:05,054 - [INFO] - from play in play-internal-execution-context-94 
Application started (Dev)

2015-03-13 23:52:29,736 - [INFO] - from play in play-internal-execution-context-94 
datasource [jdbc:h2:mem:play;DB_CLOSE_DELAY=-1] bound to JNDI as DefaultDS

2015-03-13 23:52:29,743 - [INFO] - from play in play-internal-execution-context-94 
database [default] connected at jdbc:h2:mem:play

2015-03-13 23:52:30,171 - [INFO] - from play in play-internal-execution-context-94 
Application started (Dev)

2015-03-13 23:53:15,429 - [INFO] - from play in play-internal-execution-context-101 
datasource [jdbc:h2:mem:play;DB_CLOSE_DELAY=-1] bound to JNDI as DefaultDS

2015-03-13 23:53:15,438 - [INFO] - from play in play-internal-execution-context-101 
database [default] connected at jdbc:h2:mem:play

2015-03-13 23:53:15,941 - [INFO] - from play in play-internal-execution-context-101 
Application started (Dev)

2015-03-13 23:54:48,512 - [INFO] - from play in play-internal-execution-context-101 
datasource [jdbc:h2:mem:play;DB_CLOSE_DELAY=-1] bound to JNDI as DefaultDS

2015-03-13 23:54:48,518 - [INFO] - from play in play-internal-execution-context-101 
database [default] connected at jdbc:h2:mem:play

2015-03-13 23:54:48,938 - [INFO] - from play in play-internal-execution-context-101 
Application started (Dev)

2015-03-13 23:55:59,709 - [INFO] - from play in play-internal-execution-context-101 
datasource [jdbc:h2:mem:play;DB_CLOSE_DELAY=-1] bound to JNDI as DefaultDS

2015-03-13 23:55:59,715 - [INFO] - from play in play-internal-execution-context-101 
database [default] connected at jdbc:h2:mem:play

2015-03-13 23:56:00,136 - [INFO] - from play in play-internal-execution-context-101 
Application started (Dev)

2015-03-13 23:57:16,364 - [INFO] - from play in play-internal-execution-context-101 
datasource [jdbc:h2:mem:play;DB_CLOSE_DELAY=-1] bound to JNDI as DefaultDS

2015-03-13 23:57:16,371 - [INFO] - from play in play-internal-execution-context-101 
database [default] connected at jdbc:h2:mem:play

2015-03-13 23:57:16,792 - [INFO] - from play in play-internal-execution-context-101 
Application started (Dev)

